### PR TITLE
LPS-133449 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133449

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 1]
```diff
@@ -752,4 +752,100 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-133280. Publishing a content page when staging is enabled, the changes must be reflected on the page."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:755]: Should be This is a test for LPS-130482. View the custom CSS of content page in staging site.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 2]
```diff
@@ -752,4 +752,100 @@ definition {
...
+	@priority = "4"
+	test ViewCustomChangesInContentPageWithStagingEnabled {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:757]: Should be ViewCustomCSSOfContentPageInStagingSite

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 3]
```diff
@@ -752,4 +752,100 @@ definition {
...
+		}
+
+		task ("Add a Paragraph fragment to the content page and publish") {
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Paragraph");
+
+			PageEditor.clickPublish();
+		}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:795]: Missing step ContentPagesNavigator.openViewContentPage

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 4]
```diff
@@ -752,4 +752,100 @@ definition {
...
+		task ("Add the custom CSS under the box and save") {
+			Theme.configureLookAndFeelCustomCSS(cssText = '''div.component-paragraph {background-color: blue; color:black;}''');
+
+			PortletEntry.save();
+
+			Navigator.gotoBack();
+
+			PageEditor.clickPublish();
+		}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:833]: Missing step ContentPagesNavigator.openViewContentPage

**Comment by ozysouza** [comment]
Hi @Tim-Cao, PR updated considering your comments.

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/compare/cbf20a4267807e7fe2e3082e21233ac8b339e1f7...e408a9ca2504ee7974e40d051ffc15ba9ac71d12
https://github.com/liferay-echo/liferay-portal/pull/4887

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**